lib/rules/no-regex-spaces.js

```diff
@@ -37,11 +37,11 @@ module.exports = {
          * @private
          */
         function checkRegex(node, value, valueStart) {
-            const multipleSpacesRegex = /( {2,})+?/,
+            const multipleSpacesRegex = /( {2,})+?( [+*{?]|[^+*{?])/,
```

Comment:
Honestly I'm not sure, I find it a bit hard to reason about long regexes. It seems reasonable to me though, so if the tests pass I'm fine with it.